app/views/schools/cohorts/programme_choice_no_early_career_teachers.html.erb

```diff
@@ -1,12 +1,12 @@
 <% content_for :title, "Your induction programme" %>
-<% content_for :before_content, govuk_breadcrumbs(breadcrumbs: breadcrumbs(@school, @cohort)) %>
+<% content_for :before_content, govuk_back_link(text: "Back", href: schools_dashboard_path) %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <span class="govuk-caption-l"><%= @cohort.academic_year %> cohort</span>
-    <h1 class="govuk-heading-xl govuk-!-margin-top-2">You are not expecting any early career teachers this year</h1>
-    <p class="govuk-body-l">
-      If you employ early career teachers later this year, you can <%= govuk_link_to "nominate an induction tutor", resend_email_request_nomination_invite_path %> to set up your programme.
+    <h1 class="govuk-heading-xl govuk-!-margin-top-2">You're not expecting any early career teachers this year</h1>
+    <p class="govuk-body">
+      If you employ early career teachers later this year, please contact us at <%= render MailToSupportComponent.new("contact us") %> to set up your programme.
```

Comment:
```suggestion
      If you employ early career teachers later this year, please contact us at <%= render MailToSupportComponent.new %> to set up your programme.
```

This doesn't look right 
![image](https://user-images.githubusercontent.com/40488007/133581833-9dc0881b-36f7-414f-ae2c-72a112e14915.png)

Reply:
Agreed. Now updated.